src/pages/Admin/Users/index.jsx

```diff
@@ -248,27 +343,28 @@ export default class Users extends Component {
       treeFlag,
       organizations,
       selectValue,
-      selectItem,
+      selectGroupId,
+      selectRoleId,
       selectName
     } = userStore;
     const groups = userStore.groups.toJSON();
-    //const roles = userStore.roles.toJSON();
+
     const roles = [
```

Comment:
mock data moved to external file is better

Reply:
ok, add static file store mock data.